config/notifications.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    'report' => [
+        'enabled' => env('IS_TELESCOPE_REPORT_ENABLED', false),
+        'frequency' => env('TELESCOPE_REPORT_FREQUENCY', 7),
+        'time' => env('TELESCOPE_REPORT_TIME_HOUR', 12),
+        'driver' => env('TELESCOPE_REPORT_DRIVER', 'mail'),
+        'mail_to' => env('TELESCOPE_REPORT_MAIL_TO'),
```

Comment:
let's divide it to the next structure:

```suggestion
        'driver' => env('TELESCOPE_REPORT_DRIVER', 'mail'),
        
        'drivers' => [
            'mail' => [
                'mail_to' => explode(',', env('TELESCOPE_REPORT_MAIL_TO', '')),
            ],
        
        ],
```